libs/platform/src/lib/components/form/form-group/form-group.component.ts

```diff
@@ -151,6 +153,14 @@ export class FormGroupComponent implements FormGroupContainer, OnInit, AfterCont
 
     @Input()
     compact = false;
```

Comment:
2. Keep the `compact` as undefined initially _(then we will handle it by ContentDensityService)_
```suggestion
    compact: boolean;
```